### DISCUSSION TITLE
Update Blog

### DISCUSSION THREAD

**Discussion by kdub-nz** [wtertinek/Linq2Acad#45] [General]
The last paragraph of (Part 10) says, regarding LINQ methods ;

> But diving deeper into these optimizations is beyond the scope of today’s post and a nice topic for another post.

I agree that this would be a 'nice' topic for a deep dive.

Regards,

**Comment by wtertinek** [comment]
I'm afraid I won't find the time for a blog post about this topic, but you can check out the implementation [here](https://github.com/wtertinek/Linq2Acad/blob/master/src/Linq2Acad/Enumerables/IdEnumerable.cs).
